curvefs/src/metaserver/metaserver.h

```diff
@@ -55,6 +55,7 @@ class Metaserver {
     std::shared_ptr<Trash> trash_;
     std::shared_ptr<InodeManager> inodeManager_;
     std::shared_ptr<DentryManager> dentryManager_;
+    bool enableS3Compact_;
```

Comment:
put it into `MetaserverOptions`

Reply:
i wll rewrite the s3compact option part in config later.

Reply:
fixed